ts/torch_handler/base_handler.py

```diff
@@ -180,6 +190,13 @@ def initialize(self, context):
             self.model = setup_ort_session(self.model_pt_path, self.map_location)
             logger.info("Succesfully setup ort session")
 
+        elif self.model_pt_path.endswith(".so") and TORCH_EXPORT_AVAILABLE:
```

Comment:
archiver doc strings need an update too to make it clear it also supports .so file

Reply:
not sure which one you are specifically referring to?
This doesn't talk about the format of the file
```
usage: torch-model-archiver [-h] --model-name MODEL_NAME [--serialized-file SERIALIZED_FILE] [--model-file MODEL_FILE] --handler HANDLER [--extra-files EXTRA_FILES]
                            [--runtime {python,python3}] [--export-path EXPORT_PATH] [--archive-format {tgz,no-archive,zip-store,default}] [-f] -v VERSION [-r REQUIREMENTS_FILE]
                            [-c CONFIG_FILE]
torch-model-archiver: error: the following arguments are required: --model-name, --handler, -v/--version
```